src/components/TextArea/TextArea.md

```diff
@@ -0,0 +1,51 @@
+### Examples
+
+Basic
+```js { "props": { "data-example": "basic" } }
+<TextArea
+  value="This multiline has fixed height so scrollbars may appear. This multiline has fixed height so scrollbars may appear. This multiline has fixed height so scrollbars may appear. This multiline has fixed height so scrollbars may appear. This multiline has fixed height so scrollbars may appear. This multiline has fixed height so scrollbars may appear."
+  rows={3}
+/>
+```
+
+With a description
+```js { "props": { "data-example": "withDescription" } }
+<TextArea
+  value="This multiline has fixed height so scrollbars may appear."
```

Comment:
ah i was confused on the screenshot, it doesn't look like this text reflects the example. maybe the text could read "This Textarea has a description"

Reply:
Good call. I'll do that.